### PR TITLE
SDK-323: Adding support for users to pass JVM argument using the SDK

### PR DESCRIPTION
**Pull request by wikumChamith**
## Issue I worked on

see https://openmrs.atlassian.net/browse/SDK-323

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean install` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute the above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: maven-plugin/src/main/java/org/openmrs/maven/plugins/Run.java [hunk 1]
```diff
@@ -176,6 +180,10 @@ private void runInFork(Server server) throws MojoExecutionException, MojoFailure
 
 		mavenOpts = setDebugPort(mavenOpts, server);
 
+		if(StringUtils.isNotBlank(jvmArgs)) {
```
Comment (dkayiwa) [inline maven-plugin/src/main/java/org/openmrs/maven/plugins/Run.java:183]: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle

**Comment by dkayiwa** [comment]
Will these still be available during the setup command?

**Comment by wikumChamith** [comment]
> Will these still be available during the setup command?

No, Do we need to pass JVM arguments on the setup command??

**Comment by dkayiwa** [comment]
@wikumChamith did you remember adding this to the SDK documentation?

**Comment by wikumChamith** [comment]
@dkayiwa I updated the documentation in this PR: https://github.com/openmrs/openmrs-sdk/pull/262

**Comment by dkayiwa** [comment]
Thanks! 👍

**Pull request merged**